src/pylibsshext/sftp.pyx

```diff
@@ -54,3 +54,30 @@ cdef class SFTP:
                     raise LibsshSFTPException(self, "Writing to remote file [%s] failed" % remote_file)
                 buffer = f.read(1024)
             sftp.sftp_close(rf)
+
+    def get(self, remote_file, local_file):
+        cdef sftp.sftp_file rf
+        if remote_file in ['', None]:
+            raise FileNotFoundError("ERROR: Please specify remote file name.")
+        if local_file in ['',  None]:
+            raise FileNotFoundError("ERROR: Please specify local file name.")
+        remote_file_b = remote_file.encode('utf-8')
+
+        rf = sftp.sftp_open(self._libssh_sftp_session, remote_file_b, O_RDONLY, sftp.S_IRWXU)
+        if rf is NULL:
+            raise LibsshSFTPException(self, "Opening remote file [%s] for read failed" % remote_file)
+
+        while True:
+            read_buffer = ''
```

Comment:
This will result in segmentation fault as memory is not allocated for reading bytes to be stored.